wzdx/standard_to_enhanced/icone_translator.py

```diff
@@ -47,24 +48,28 @@ def parse_icone_arguments():
 
 def wzdx_creator(message, info=None):
     if not message:
+        print('returning None 50')
```

Comment:
Should we swap all these print statements to logging?

Reply:
removed unnecessary print statements